src/erlware_commons.app.src

```diff
@@ -3,9 +3,9 @@
               {vsn,"0.18.0"},
               {modules,[]},
               {registered,[]},
-              {applications,[kernel,stdlib]},
-              {contributors,["Eric Merritt","Tristan Sloughter",
-                             "Jordan Wilberding","Martin Logan"]},
+              {applications,[kernel,stdlib,cf]},
+              {maintainers,["Eric Merritt","Tristan Sloughter",
+                            "Jordan Wilberding","Martin Logan"]},
```

Comment:
Not sure why, but it saddens me I had an errant whitespace in front of my name this whole time :(